src/__tests__/ModuleList/Module/SubModule/SubModule.test.jsx

```diff
@@ -12,18 +13,33 @@ const defaultProps = {
   type: 'article',
 }
 
+const valuesProps = {
+  title: 'Article',
+  progress: 99,
+  id: 1,
+  moduleId: 2,
+  type: 'value',
+}
+
 describe('The sub modules', () => {
   it('should render and match previous snapshots', () => {
-    const tree = renderer.create(
+    const tree = renderTreeWithTheme(
       <SubModule {...defaultProps} />
     ).toJSON();
     expect(tree).toMatchSnapshot();
   });
   it('should create a link to the submodule page', () => {
-    const { moduleId, type, id, title } = defaultProps;
-    const { getByText } = render(<SubModule {...defaultProps} />);
+    const { type, id, title } = defaultProps;
+    const { getByText } = renderWithTheme(<SubModule {...defaultProps} />);
     const link = getByText(title);
 
-    expect(link.getAttribute('href')).toBe(`/module/${moduleId}/${type.toString()}/${id}`)
+    expect(link.getAttribute('href')).toBe(`/module/0/${type.toString()}/${id}`)
```

Comment:
Oh! Maybe we're hardcoding `0` for now?

Reply:
Yes. I hardcoded that for now. But I think you're right that it should just be `moduleId` instead of being hardcoded!

Reply:
Ok, then everything looks good!
What do think about the additional tasks for the lengths of the progress bars and the modules remaining unfolded?

Reply:
Yeah, I think I'll just make those additional tickets!

Reply:
Sounds good! I'll approve!